.github/workflows/node.js.yml

```
@@ -16,7 +16,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [16.x, 18.x, 20.x]
+        node-version: [16.x, 18.x, 20.x, 21.x]
```

```suggestion
        node-version: [18.x, 20.x, 21.x]
```
consider removing 16.x since its active support has ended on 18th Oct 2023 and its security support has ended on 11th Sep 2023. We shouldn't support it anymore.